packages/components/icon/src/icon.css

```diff
@@ -24,10 +24,69 @@ svg {
 .icon-wrapper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Check this potential box model size issue. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Abox-model)